rts/Game/GameHelper.cpp

```diff
@@ -1253,6 +1253,9 @@ float CGameHelper::GetBuildHeight(const float3& pos, const UnitDef* unitdef, boo
 }
 
 
+/**
+ * @param feature Deprecated, just pass nullptr.
```

Comment:
really?

Reply:
I believe so. Although I probably should have written "unused" not "deprecated".

The param is set to null immediately and not read. I only added the comment because I was confused by some code calling it.

https://github.com/beyond-all-reason/spring/blob/5931c8253a83aa694ff765524e2e69436f17d0eb/rts/Game/GameHelper.cpp#L1271

Reply:
That's because it's an out param. Some inner functions set it and its value after the function exits is meaningful in the caller.